### PR TITLE
fix textSplitter.createDocuments~ missing await

### PR DESCRIPTION
**Pull request by jacobrosenthal**
More from discord @nfcampos 
https://github.com/hwchase17/langchainjs/pull/328

### CODE REVIEW

File: docs/docs/modules/llms/overview.md [hunk 1]
```diff
@@ -45,21 +45,22 @@ const model = new OpenAI({ concurrency: 5 });
 Especially when using an agent, there can be a lot of back-and-forth going on behind the scenes as a LLM processes a chain. For agents, the response object contains an intermediateSteps object that you can print to see an overview of the steps it took to get there. If that's not enough and you want to see every exchange with the LLM, you can use the LLMCallbackManager to write yourself custom logging (or anything else you want to do) as the model goes through the steps:
 
 ```typescript
-import { LLMCallbackManager } from "langchain/llms";
+import { LLMResult } from "langchain/schema";
```
Comment (nfcampos) [inline docs/docs/modules/llms/overview.md:49]:
```suggestion
import { CallbackManager } from "langchain/callbacks";

```

File: examples/src/chains/llm_chain_stream.ts [hunk 2]
```diff
@@ -5,7 +5,7 @@ import { CallbackManager } from "langchain/callbacks";
 
 export const run = async () => {
   const manager = CallbackManager.fromHandlers({
-    async handleLLMNewToken(token) {
+    async handleLLMNewToken(token: string) {
```
Comment (nfcampos) [inline examples/src/chains/llm_chain_stream.ts:8]: This type is inferred by Typescript, so not terribly useful?

Comment (jacobrosenthal) [reply examples/src/chains/llm_chain_stream.ts:8]: my linter always had it red and it was annoying me shrug.

Comment (nfcampos) [reply examples/src/chains/llm_chain_stream.ts:8]: oh strange, lets keep your change then

**Pull request merged**